index.js

```diff
@@ -104,6 +143,14 @@ console.log('hello world')
 
 // TODO: ADD CODE HERE
 
+function stringToUpper (str="") {
```

Comment:
nice! this `str=""` part is a good way to provide a default value. so if you  didn't pass in a value on line 152, it would just default to the value of empty string for `str`.